MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewController.swift

```diff
@@ -8,10 +8,25 @@
 import UIKit
 
 class MovieDetailsViewController: UIViewController {
-    private let movieDetailsViewModel = MovieDetailsViewModel()
 
+    // MARK: - IBOutlets
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var posterImageView: UIImageView!
+
+    // MARK: - Variables
+    var viewModel = MovieDetailsViewModel()
```

Comment:
Please use dependency injection for the repo like you did in the other files below

Reply:
will do, I was going to add the same changes we did to the others as I touched them again but I'll add them now 🤝